### ISSUE TITLE
Rework the landing page getguaka.com

### ISSUE THREAD

**Issue by oarrabi** [oarrabi/Guaka#69]
The current website lacks some of the information about what libraries are part of Guaka
This website http://piotrmurach.github.io/tty/ does a great job at highlighting its component.  Maybe we can do something similar to it.

**Comment by goyox86** [comment]
Yeah but I think our case is a bit different. Because in their case they designed the library and modularized that way to provide a better UX. For us the case was different we just had to do it because such features don't exist today at least in a standard library. is just that we were good citizens and modularized the those things so they could be used on their own. SO I think the approach we should  take it's something along the lines of: "Oh BTW Guaka is built on top of these libraries, you might want to take a look at them..." And then we showcase those.

What do you think @oarrabi ?

**Comment by oarrabi** [comment]
If we want to expand we can make coloring part of Guaka, choices, yes/now questions also part of it.
We can then implement progress bar, table etc..

All these can become part of guaka!
